src/kibana/plugins/visualize/editor/styles/editor.less

```diff
@@ -153,6 +153,14 @@
         font-weight: bold;
       }
 
+      &-growing {
```

Comment:
Not sure on this class naming? Also, this really isn't any kind of `vis-editor-agg-header`